### PR TITLE
making first progress

### PR DESCRIPTION
**Pull request by neacatsu**

### CODE REVIEW

File: src/app/MainMenu.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
+const MENU = {
+            mainPage: {name: 'Main Page', link: ''},
+            randomRecipe: {name: 'Random Recipe', link: ''},
+            nutritionGame: {name: 'Nutrition Game', link: ''},
+            calculator: {name: 'Calculator', link: ''},
+            shoppingList: {name: 'Shopping List', link: './list.html'}
+          }
+
+
+export const MainMenu = (activePage) => {
```
Comment (radekwojpl2) [inline src/app/MainMenu.js:10]: You have Kinga's changes, watch out for such a situation, you can push code to production which is not ready for it

File: src/app/ShoppingList.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                })
+            })
+        });      
+}
+
+function clearSingleIngedient(arr1, arr2) {
+    arr2.forEach((el) => {
+        el.addEventListener('click', function () {
+            const index = arr2.indexOf(el);
+            arr2.splice(index, 1);
```
Comment (radekwojpl2) [inline src/app/ShoppingList.js:71]: change those names to appropriate ones, these names don't tell anything, what is arr1 and arr2?

File: src/app/ShoppingList.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+                    clearSingleIngedient(res, arr);
+                    clearShoppingList();
+                })
+            })
+        });      
+}
+
+function clearSingleIngedient(arr1, arr2) {
+    arr2.forEach((el) => {
```
Comment (radekwojpl2) [inline src/app/ShoppingList.js:68]: don't use shortcuts, it is better to use the whole name than something like el

File: list.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
+    <title>CodersCamp2020 | Star Wars API QUIZ</title>
```
Comment (radekwojpl2) [inline list.html:7]: Star Wats Api Quiz ?????

File: src/app/MainMenu.js [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+  //change visibility of menu for mobile
+  const changeVisibilityForMenu = () => {
+    navigationBox.classList.toggle('navigationBoxHidden');
+  }
+
+  toggleBtn.addEventListener('click', e => {
+      changeVisibilityForMenu();
+      clearSearchInfo();
+  })
+}
```
Comment (mateugazi) [inline src/app/MainMenu.js:195]: Why there is a Marta's file?

File: list.html (hunk 4 above)
Comment (neacatsu) [reply list.html:7]: Changed.

File: src/app/ShoppingList.js (hunk 3 above)
Comment (neacatsu) [reply src/app/ShoppingList.js:68]: All names renamed

**Review by radekwojpl2 — COMMENTED** [review]
Approved

File: list.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+            </div>
+        </div>
+    </div>
+        
+        
+        
+</div> 
+
+<script src="src/index.js"></script>
+<script src="src/app/ShoppingList.js"></script>
```
Comment (mateugazi) [inline list.html:38]: module

**Review by MaDud — APPROVED** [review]

File: src/app/ShoppingList.js [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+            let output = ``;
+            data.forEach(element => {
+                output += ` <li><button data-id='${element.id}'>${element.title}</li>`; 
+            });
+            document.querySelector('.dish').innerHTML = output;
+            
+            document.querySelectorAll('.dish li button').forEach((e) => {
+                e.addEventListener('click', function() {
+                    console.log(e.textContent);
```
Comment (JotGabiga) [inline src/app/ShoppingList.js:19]: Do we need this here? :-)

File: src/app/ShoppingList.js [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+        .then(data => {
+            let output = ``;
+
+            data.ingredients.forEach(element => {
+                output += `<li>${element.name} <button class="fa fa-plus-circle"></button></li>`; 
+            });
+            document.querySelector('.ingredients').innerHTML = output;
+
+            const arrayIngredients = document.querySelectorAll('.ingredients li');
+            console.log(arrayIngredients);
```
Comment (JotGabiga) [inline src/app/ShoppingList.js:51]: Same here

**Review by JotGabiga — APPROVED** [review]

File: src/app/ShoppingList.js (hunk 7 above)
Comment (neacatsu) [reply src/app/ShoppingList.js:19]: console.log? One deleted, leaving one for the project requirements

**Review by Yoruko90 — APPROVED** [review]

**Review by JotGabiga — APPROVED** [review]

**Pull request merged**